features/features.go

```diff
@@ -0,0 +1,96 @@
+//go:generate stringer -type=FeatureFlag
+
+package features
+
+import (
+	"expvar"
+	"fmt"
+	"sync"
+)
+
+type FeatureFlag int
+
+const (
+	unused                    FeatureFlag = iota
+	DNSAllowLoopbackAddresses             // Common
+	CheckMalformedCSR                     // WFE
+	DoNotForceCN                          // RA, CA
+	ReuseValidAuthz                       // RA
+	EnableMustStaple                      // CA
+	AcceptRevocationReason                // WFE
+)
+
+// List of features and their default value
```

Comment:
For things like the flag in WFE for hoisting a SAN to CN where we want the behavior to be default on and has to be explicitly disabled.